settings/src/test/java/org/odk/collect/settings/validation/JsonSchemaSettingsValidatorTest.kt

```diff
@@ -4,12 +4,7 @@ import org.hamcrest.MatcherAssert.assertThat
 import org.hamcrest.Matchers.equalTo
 import org.junit.Test
 
-class JsonSchemaSettingsValidatorTest {
-
-    @Test
-    fun `returns true when json is valid based on schema`() {
-        val validator = JsonSchemaSettingsValidator {
-            """
+private const val SCHEMA = """
```

Comment:
Can we move this below the test class, so you don't have to scroll down to get to the tests themselves here?